source/validate_atomics.cpp

```diff
@@ -23,6 +23,89 @@
 
 namespace libspirv {
 
+// Returns number of '1' bits in a word.
+uint32_t CountSetBits(uint32_t word) {
```

Comment:
FYI this is also known as "population count" or popcount which is sometimes a builtin in some compilers. 

But this is fine.  :-)